openpype/lib/avalon_context.py

```diff
@@ -494,6 +497,18 @@ def get_workfile_template_key(
     return default
 
 
+def _get_system_general_data():
```

Comment:
I would convert it back to public function and use it in `openpype/plugins/publish/collect_anatomy_context_data.py` and `openpype/tools/workfiles/app.py`.